lib/smart_answer/calculators/find_coronavirus_support_calculator.rb

```diff
@@ -38,10 +38,10 @@ class FindCoronavirusSupportCalculator
         calculator.needs_help_with?("getting_food") && calculator.get_food != "yes"
       },
       have_you_been_made_unemployed: lambda { |calculator|
-        calculator.needs_help_with?("being_unemployed") && calculator.have_you_been_made_unemployed != "no"
+        calculator.needs_help_with?("being_unemployed") && calculator.have_you_been_made_unemployed?
```

Comment:
Nit: how come the extra method?

```suggestion
        return false if have_you_been_made_unemployed.blank?
        
        calculator.needs_help_with?("being_unemployed") && calculator.have_you_been_made_unemployed != "no"
```

Reply:
I think `SECTIONS` is over-complicated and was trying to avoid further complicating it.